### DISCUSSION TITLE
How skip behavior of destroying the previous file if it is updated

### DISCUSSION THREAD

**Discussion by mariaLauraDev** [shrinerb/shrine#712] [General]
In my application I have a class that inherits Shrine from the gem and I don't want it to have the behavior of destroying the previous file if it is updated, because this generates corruption in the path of the previous files that are still requested by my system, because it is stored in cache. Is there any method or configuration that I can declare in my class to avoid the behavior of the following screenshot?

![image](https://github.com/user-attachments/assets/267b967c-bdb5-4e6e-87ba-2554db357f9d)

My class that inherits shrine

``
class FontUploader < Shrine
end
``

**Comment by janko** [comment]
There isn't a direct configuration method, but you could override the method and make it a no-op:

```rb
class FontUploader < Shrine
  class Attacher
    def destroy_previous
      # don't delete replaced attachments
    end
  end
end
```

**Comment by mariaLauraDev** [comment]
Thanks!! That's works perfectly
